Update README.md (#44)
OWASP/www-project-mobile-security-testing-guide: README.md

# Mobile Security Guide Website repo

This repo has the files for https://owasp.org/www-project-mobile-security-testing-guide/. 

A few things to keep in mind:

>  - For changes: file a [PR](https://github.com/OWASP/www-project-mobile-security-testing-guide/pulls) (Fork this repo, create changes on your local fork and then do a PR, like described in [the MSTG](https://github.com/OWASP/owasp-mstg/blob/master/CONTRIBUTING.md#how-to-set-up-your-contributor-environment), just make sure you use the right url (github.com/OWASP/www-project-mobile-security-testing-guide instead of github.com/OWASP/owasp-mstg/)

Reviewer: ```suggestion
 - For changes: file a [PR](https://github.com/OWASP/www-project-mobile-security-testing-guide/pulls) by forking this repo, creating changes on your local fork and then opening a PR, like described in the [Contributing page](https://github.com/OWASP/owasp-mstg/blob/master/CONTRIBUTING.md#how-to-set-up-your-contributor-environment) in the MSTG repo. Please make sure you use the right URL ("github.com/OWASP/www-project-mobile-security-testing-guide" instead of "github.com/OWASP/owasp-mstg").
```

>  - If you see an issue, but do not have the time to file a PR: file an [issue](https://github.com/OWASP/www-project-mobile-security-testing-guide/issues)

Reviewer: ```suggestion
 - If you notice that something's wrong or could be improved, but do not have the time to open a PR: file an [issue](https://github.com/OWASP/www-project-mobile-security-testing-guide/issues).
```

>  - If your issue is not picked up timely, contact us on Slack (See the [MSTG repo](https://github.com/OWASP/owasp-mstg/) for more details).

Reviewer: ```suggestion
 - If your issue is not picked up timely, contact us on Slack (see the [MSTG repo](https://github.com/OWASP/owasp-mstg/) for more details).
```

>  - Please adhere to the [Code of Conduct](https://github.com/OWASP/owasp-mstg/blob/master/CODE_OF_CONDUCT.md) when filing issues, filing PRs or reviewing them.

Reviewer: ```suggestion
 - Please read the [Code of Conduct](https://github.com/OWASP/owasp-mstg/blob/master/CODE_OF_CONDUCT.md) carefully and adhere to it at any time.
```
